src/createXcmTypes/ParaToSystem.ts

```diff
@@ -44,21 +45,27 @@ export const ParaToSystem: ICreateXcmType = {
 	 */
 	createBeneficiary: (accountId: string, xcmVersion?: number): XcmDestBenificiary => {
 		if (xcmVersion == 2) {
+			const X1 = isEthereumAddress(accountId)
```

Comment:
Can a system parachain have a `ethereum` address?

Reply:
ahh, good point. Im guessing that's not possible currently. Will revert changing this direction

Reply:
Sounds good! I wasn't sure if there is a possible type of system parachain that has an ethereum address, but probably best to be safe, and remove it.